src/com/divergent/clinicmanagmentsystem/Appoinment.java

```diff
@@ -87,31 +92,27 @@ public static void appoinmentPanel() {
 	public static void showAllAppoinment() {
 		try {
 			Class.forName("com.mysql.cj.jdbc.Driver");
-			Connection con = DriverManager.getConnection(ClinicDatabase.url, ClinicDatabase.user,
-					ClinicDatabase.password);
+			Connection con = DriverManager.getConnection(ClinicDatabase.URL, ClinicDatabase.USERNAME,
+					ClinicDatabase.PASSWORD);
 			Statement st = con.createStatement();
 			String sql = "select * from appoinment";
 			ResultSet rs = st.executeQuery(sql);
-			if (!rs.next()) {
-				System.out.println("No Record Is Found!\n");
-				doctor.printDoctorOptions();
-			} else {
-				System.out.println(
-						"\n*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*Appointmenr Patient Data*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*\n");
-				int appoin_id = rs.getInt(1);
-				String p_name = rs.getString(2);
-				String d_name = rs.getString(3);
+			System.out.println(
+					"\n*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*Appointmenr Patient Data*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*\n");
+			while (rs.next()) {
+				int appoinid = rs.getInt(1);
+				String pname = rs.getString(2);
+				String dname = rs.getString(3);
 				String problem = rs.getString(4);
 				String appinDate = rs.getString(5);
 				Date currentDate = rs.getDate(6);
-				String d_id = rs.getString(7);
-				String p_id = rs.getString(8);
-				System.out.printf("%6s  %15s  %15s  %20s  %12s  %12s  %5s  %5s\n", appoin_id, p_name, d_name, problem,
-						appinDate, currentDate, d_id, p_id);
-
-				System.out.println(
-						"*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*\n");
+				String did = rs.getString(7);
+				String pid = rs.getString(8);
+				System.out.printf("%6s  %15s  %15s  %20s  %12s  %12s  %5s  %5s\n", appoinid, pname, dname, problem,
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.